PowerPointLabs/PowerPointLabs/TimerLab/TimerLabPaneWPF.xaml.cs

```diff
@@ -438,28 +438,20 @@ private void WidthSlider_ValueChanged(object sender, RoutedPropertyChangedEventA
                 var timerBody = GetShapeByName(TimerLabConstants.TimerBodyId);
                 float increment = value - timerBody.Width;
 
-                if (increment == 0)
-                {
-                    ReformMarkersIfMissing();
-                }
-                else
-                {
-                    var lineMarkerGroup = GetLineMarkerGroup();
-                    var timeMarkerGroup = GetShapeByName(TimerLabConstants.TimerTimeMarkerGroupId);
-                    timerBody.Select();
-                    if (lineMarkerGroup != null)
```

Comment:
Ok, i changed some parts and hopefully all null cases are covered.
Also fixed the exceptions, thanks!